deploy/olm-catalog/ibm-iam-operator/3.5.0/ibm-iam-operator.v3.5.0.clusterserviceversion.yaml

```diff
@@ -585,7 +585,7 @@ spec:
   maintainers:
   - email: support@ibm.com
     name: IBM Support
-  maturity: alpha
+  maturity: stable-v1
```

Comment:
Cannot use `stable-v1` in here, OLM doesn't support this value, you can use `stable`

Reply:
okay